tools/autograd/templates/VariableType.cpp

```diff
@@ -312,11 +312,12 @@ void VariableType::s_copy(const Tensor & src, Tensor & dst) const {
   auto flags = compute_flags({ dst, src });
   flags.requires_grad &= isFloatingPoint(dst.type().scalarType());
   if (flags.requires_grad) {
-    // TODO: handle type conversions
+    // TODO: handle device movement
     grad_fn = std::make_shared<CopyBackwards>();
     grad_fn->is_executable = true;
-    grad_fn->next_functions = compute_next_functions({ dst, src });
+    grad_fn->next_functions = compute_next_functions({ src, dst });
```

Comment:
Why flip the order?

Reply:
That's the order of the function call (maybe we should flip that, but these should match up imo).

Reply:
Yeah, that makes sense. I was just curious why did you change it